packages/date-picker/src/vaadin-date-picker.js

```diff
@@ -161,34 +160,21 @@ class DatePicker extends DatePickerMixin(InputControlMixin(ThemableMixin(Element
       <vaadin-date-picker-overlay
         id="overlay"
         fullscreen$="[[_fullscreen]]"
-        theme$="[[__getOverlayTheme(_theme, _overlayInitialized)]]"
```

Comment:
Do you know why the theme was only set after the overlay was initialized?

Reply:
This was done in https://github.com/vaadin/vaadin-date-picker/pull/652 to fix an [error](https://github.com/vaadin/vaadin-date-picker/issues/651) thrown when setting theme.
No longer needed as this PR removes usage of `disable-upgrade` Polymer API.